src/app/_components/_sharedcomponents/Cards/LeaderBaseCard/LeaderBaseCard.tsx

```diff
@@ -13,116 +13,148 @@ import { s3CardImageURL } from "@/app/_utils/s3Utils";
+        backgroundImage: `url(${s3CardImageURL(card)})`,
+        backgroundSize: "contain",
+        backgroundPosition: "center",
+        width: "9.5vw",
+        height: "13vh",
+        backgroundRepeat: "no-repeat",
+        textAlign: "center" as const,
+        color: "white",
+        display: "flex",
+        cursor: "pointer",
+        position: "relative" as const,
+        mb: "10px",
+    } : {
+        backgroundColor: "#00000040",
+        backgroundSize: "contain",
+        backgroundPosition: "center",
+        width: "9.5vw",
+        height: "13vh",
+        backgroundRepeat: "no-repeat",
+        textAlign: "center" as const,
+        color: "white",
+        display: "flex",
+        cursor: "pointer",
+        position: "relative" as const,
+        mb: "10px",
+    };
 
-	const typographyStyle = {
-		fontFamily: "var(--font-barlow), sans-serif",
-		fontWeight: "400",
-		fontSize: "1.6em",
-	};
+    const damageStyle = {
+        fontWeight: "600",
+        fontSize: "2em",
+        color: "hotpink",
+    };
 
-	const damageStyle = {
-		fontWeight: "600",
-		fontSize: "2em",
-		color: "hotpink",
-	};
+    const titleTypographyStyle = {
+        fontFamily: "var(--font-barlow), sans-serif",
+        fontWeight: "600",
+        fontSize: "1.5em",
+        marginBottom: isLobbyView ? 0 : "0.5em",
+        textAlign: "left",
+        color: "white",
+    };
+    const titleTypographyStyleOpponent = {
+        fontFamily: "var(--font-barlow), sans-serif",
+        fontWeight: "600",
+        fontSize: "1.5em",
+        marginBottom: "10px",
+        textAlign: "left" as const,
+        color: "white",
+        opacity: "15%",
+    }
 
-	const titleTypographyStyle = {
-		fontFamily: "var(--font-barlow), sans-serif",
-		fontWeight: "600",
-		fontSize: "1.5em",
-		marginBottom: isLobbyView ? 0 : "0.5em",
-		textAlign: "left",
-		color: "white",
-	};
+    //the title of the deck i believe
+    const redBoxStyle = {
+        position: "absolute",
+        bottom: "10px",
+        left: "50%",
+        transform: "translateX(-50%)",
+        backgroundColor: "red",
+        borderRadius: "4px",
+        p: "4px 8px",
+    };
 
-	//the title of the deck i believe
-	const redBoxStyle = {
-		position: "absolute",
-		bottom: "10px",
-		left: "50%",
-		transform: "translateX(-50%)",
-		backgroundColor: "red",
-		borderRadius: "4px",
-		p: "4px 8px",
-	};
+    const redBoxTypographyStyle = {
+        color: "white",
+        fontFamily: "var(--font-barlow), sans-serif",
+        fontWeight: "600",
+        fontSize: "1em",
+    };
 
-	const redBoxTypographyStyle = {
-		color: "white",
-		fontFamily: "var(--font-barlow), sans-serif",
-		fontWeight: "600",
-		fontSize: "1em",
-	};
+    const { sendGameMessage } = useGame();
 
-	const { sendGameMessage } = useGame();
+    return (
+        <Box>
+            {/* Show title above the card if in lobby view and variant is leader */}
+            {variant === "leader" && isLobbyView && title && (
```

Comment:
Similar to GameCard let's try to make thuis one more location agnostic. This title should live in the Lobby component since it is specific to that layout. For the onclick we can do a similar thing to the game card and it may be a good idea to add a boolean `disabled` prop (perhaps to GameCard) as well that would allow us to disable onclick events for that card.